packages/p2p-v2/src/components/AdvertsTableRow/AdvertsTableRow.tsx

```diff
@@ -100,15 +100,16 @@ const AdvertsTableRow = memo((props: TAdvertsTableRowRenderer) => {
                             <div className='flex items-center'>
                                 {hasRating ? (
                                     <>
-                                        <Text className='lg:mr-0 mr-[-1.2rem]' color='less-prominent' size='xs'>
+                                        <Text className='lg:mr-0' color='less-prominent' size='xs'>
                                             {ratingAverageDecimal}
                                         </Text>
                                         <StarRating
+                                            allowHalfIcon
                                             isReadonly
                                             ratingValue={Number(ratingAverageDecimal)}
                                             starsScale={isMobile ? 0.7 : 0.9}
                                         />
-                                        <Text className='lg:ml-1 ml-[-1rem]' color='less-prominent' size='xs'>
+                                        <Text className='lg:ml-[-0.5rem] ml-[-2.5rem]' color='less-prominent' size='xs'>
```

Comment:
Could you try this? Usually negative is prefixed at the start of the class name
```suggestion
                                        <Text className='lg:-ml-[0.5rem] -ml-[2.5rem]' color='less-prominent' size='xs'>
```

Reply:
i checked and it does work butttt i still feel the other way around is more readable than -ml, cause you as a coder will know the value is negative since its wrapped inside [ ]